tests/resources/test_common.py

```diff
@@ -21,35 +22,20 @@ def __init__(self, id: str, **attributes):
         """
         super().__init__(id=id, **attributes)
 
-    @property
-    def id(self) -> str:
-        """
-        ID of the Stream
-
-        """
-        return self._attributes.get("id")
-
 
 class PatientItem(ItemBase):
     """
     Test ItemBase class that represents a Patient
 
     """
 
-    def __init__(self, id: str, **attributes):
+    def __init__(self, id: str, name: str = "", **attributes):
         """
         Initializes PatientItem
 
         """
-        super().__init__(id=id, **attributes)
-
-    @property
-    def id(self) -> str:
-        """
-        ID of the Patient
-
-        """
-        return self._attributes.get("id")
+        self.name = name
+        super().__init__(id=id, name=name, **attributes)
```

Comment:
yeah... I thought about it, but punted -- partially because I think a lot of these tests could be deleted, now that the actual implementations exist. I don't think it's harmful to leave as is, but I'll create a backlog ticket to refactor this.